README.md

```diff
@@ -1,16 +1,8 @@
-## Titanium CLI
+# Titanium CLI
 
-> [Titanium CLI](https://github.com/appcelerator/titanium) is a
-[Command Line Tool](http://en.wikipedia.org/wiki/Command-line_interface)
-for creating and building Titanium Mobile applications and modules. It's
-open-source and easy to use. [We've](https://github.com/appcelerator)
-designed Titanium to be suitable for command line beginners, but still be
-powerful and extensible enough for production usage.
+> [Titanium CLI](https://github.com/tidev/titanium) is a [Command Line Tool](http://en.wikipedia.org/wiki/Command-line_interface) for creating and building Titanium Mobile applications and modules. It's open-source and easy to use. [We've](https://github.com/tidev) designed Titanium to be suitable for command line beginners, but still be powerful and extensible enough for production usage.
```

Comment:
I think we can remove the wikipedia link and the link to tidev.

Reply:
is removed